src/views/misc/Terms.vue

```diff
@@ -0,0 +1,127 @@
+    <p>These Terms of Use will be interpreted under and governed by the laws of California (United States) excluding any laws that might direct the application of the laws of another jurisdiction. Each party waives all rights, if any, arising under the United National Convention on Contracts for the International Sale of Goods, and all rights of application or appeal, if any, to non-U.S. courts in connection with any question of law. Any litigation arising from these Terms of Use or your use of MorphoBank or the Services will be undertaken in California courts or Federal Courts located in California, and each party consents to such jurisdiction. Each party shall be entitled, without waiving any other rights or remedies, to injunctive or other equitable relief.</p>
+    <p>Phoenix has a property ownership as well as copyright rights covering MorphoBank, the Services and the Materials. You hereby acknowledge that the business of Phoenix is dependent upon maintaining Phoenix’s proprietary rights in and control of the MorphoBank, the Services and the Materials, and that any unauthorized reproduction, distribution, or furnishing of access thereto constitutes a fundamental breach of these Terms of Use, in which event, Phoenix will be entitled to immediate injunctive relief and the immediate removal of all electronic copies of the Materials held by you (or your unauthorized transferees) without prejudice to any other rights or remedies to which Phoenix may be entitled.</p>
+
+    <h2>10. Changes to Terms of Use</h2>
+    <p>These Terms of Use may be amended by Phoenix from time to time at our sole discretion, without notice to you or other users. Please periodically review the controlling version of these Terms of Use available here. By continuing to access or use MorphoBank or the Services subsequent to Phoenix making available an amended version of these Terms of Use, you acknowledge, agree and consent to such amendment.</p>
+    <h2>11. Indemnity</h2>
+    <p>You will indemnify, save, hold harmless and defend Phoenix from and against any and all claims, damages, demands, actions, judgments, lawsuits, proceedings, assessments, liabilities, losses, penalties, costs and expenses (including, without limitation, reasonable attorneys' fees, costs and expenses), whether or not subject to litigation (collectively, "Indemnified Claims"), incurred by any Indemnified Party in connection with, by reason of, or arising directly or indirectly out of (i) use of MorphoBank, the Services or the Materials, including without limitation the upload of any Data, by you or any person acting by or through you (including, without limitation, by utilizing your login information), or (ii) the breach of these Terms of Use by you or any person acting by or through you (including, without limitation, by utilizing your login information).</p>
```

Comment:
Can you split the conditions into multiple lines (i,ii,iii,iii) so it is consistent with the format above?